README.md

```diff
@@ -0,0 +1,69 @@
+# OCP DNSNameResolver
+
+## Name
```

Comment:
I don't quite get the naming of this chapter. I expected some context about the name of the plugin but it starts explaining some details about what the plugin does. Seems rather like the beginning of the `Description` chapter.

Reply:
Also I think that the first thing we need to say is that this is a CoreDNS plugin.

Reply:
Any CoreDNS plugin Readme follows the format of Name, Description, etc. The Name section gives a brief overview of what the plugin does. You can refer to https://coredns.io/explugins/ to see how the Name section of the external plugins are used to display the overview of the plugins. I kind of followed the documentation of these plugins to write these Readme file.